src/api/forms/service.js

```diff
@@ -66,11 +70,14 @@ export async function createForm(metadata, author) {
 
   // Create the metadata document
   const { insertedId: _id } = await formMetadata.create(document)
+  const createdForm = mapForm({ ...document, _id })
```

Comment:
We should probably rename the param `metadata` → `metadataInput`

Because this new const is the _actual_ output `metadata`

```suggestion
  const metadata = mapForm({ ...document, _id })
```

Reply:
Done